poetry.lock

```diff
@@ -2,38 +2,39 @@
 
 [[package]]
 name = "annotated-types"
-version = "0.5.0"
+version = "0.7.0"
```

Comment:
Looks like a lot of dependencies were updated, did you maybe run `poetry lock`  without the `--np-update` flag? See: https://python-poetry.org/docs/cli/#options-11

Reply:
I misunderstood your initial suggestion to update the poetry file, as I needed to run the update command, oops. I've never worked with poetry before to be honest. reverted and changed to the  `poetry lock --no-update`  command

Reply:
No problem, I will merge this and create a release shortly :)